core/trino-main/src/main/java/io/trino/execution/CreateRoleTask.java

```diff
@@ -21,6 +21,7 @@
 import io.trino.spi.security.TrinoPrincipal;
```

Comment:
It would be nice to add some tests to `TestHiveRoles` to make sure that this new syntax is effective.

Reply:
I added more tests